sdk/src/clients/consumer.rs

```diff
@@ -675,9 +793,9 @@ impl IggyConsumerBuilder {
             partition: partition_id,
             polling_strategy: PollingStrategy::next(),
             batch_size: 1000,
-            auto_commit: AutoCommit::IntervalOrAfter(
+            auto_commit: AutoCommit::IntervalOrWhen(
                 IggyDuration::from(SEC_IN_MICRO),
```

Comment:
maybe it's good idea to create certain constants that can be easily created for example IggyDuration::Second(), etc...

Reply:
Done